src/server/config.rs

```diff
@@ -181,6 +183,12 @@ impl Config {
             ));
         }
 
+        if self.grpc_stream_initial_window_size.0 > i32::MAX as u64 {
```

Comment:
It's a `ReadableSize` so that we can parse it from `15M`, `10G`, ... Use `i32` directly will change it. Maybe we should change `ReadableSize` to `ReadableSize<I>`?

Reply:
I like this idea but lets do that in another ticket. :) Maybe you can open a follow up?